meson.build

```diff
@@ -61,6 +61,11 @@ panel_tool_a = static_library(
      'tools/src/dbus_call.cpp',
      include_directories: ['include', 'tools/include']
 )
+logger_a = static_library(
```

Comment:
why _a at the end of variable name?

Reply:
> why _a at the end of variable name?

I think, she is following the existing pattern , as
panel_app_a = static_library(
panel_tool_a = static_library(